cvmfs/monitor.h

```diff
@@ -42,6 +43,15 @@ class Watchdog : SingleCopy {
   ~Watchdog();
   void Spawn(const std::string &crash_dump_path);
 
+  /**
+   * All signals overwritten by the
+  */
+  static int all_signals[13];
```

Comment:
Indicate as a global:
```suggestion
  static int g_all_signals[13];
```

Reply:
// The watchdog is not supposed to receive signals. If it does, report it.

I therefore suggest renaming it to `g_suppressed_signals`